src/assignments/pavanavvari/Asignment1.java

```diff
@@ -5,11 +5,26 @@ public class Asignment1 {
 	// Write a reusable program to check the number prime or not
 
 	public boolean checkPrime(int num) {
+		int ctr=0;
+		while(num>=1)
+		{
```

Comment:
Chnage the logic,as it is not checking the prime number

Reply:
Logic updated and verified.